src/Microsoft.Health.Dicom.Functions/Management/PurgeOrchestrationInstancesHistory.cs

```diff
@@ -0,0 +1,88 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Health.Dicom.Functions.Management
+{
+    public class PurgeOrchestrationInstancesHistory
```

Comment:
nit: let's call it `PurgeOrchestrationInstanceHistory` instead (singular `Instance`)